test/manifoldc_test.cpp

```diff
@@ -23,19 +23,35 @@ TEST(CBIND, warp_translation) {
     ManifoldVec3 v = {x + 15.0f, y, z};
     return v;
   };
+  float *context = (float *)malloc(1 * sizeof(float));
+  context[0] = 15.0f;
+  ManifoldVec3 (*warpcontext)(
+      float, float, float, void *) = [](float x, float y, float z, void *ctx) {
+    ManifoldVec3 v = {x + ((float *)ctx)[0], y, z};
+    return v;
+  };
   ManifoldManifold *sphere = manifold_sphere(malloc(sz), 1.0f, 100);
   ManifoldManifold *trans = manifold_translate(malloc(sz), sphere, 15., 0., 0.);
   ManifoldManifold *warped = manifold_warp(malloc(sz), sphere, warp);
   ManifoldManifold *diff = manifold_difference(malloc(sz), trans, warped);
+  ManifoldManifold *warpedcontext =
+      manifold_warp_context(malloc(sz), sphere, warpcontext, context);
+  ManifoldManifold *diffcontext =
+      manifold_difference(malloc(sz), trans, warped);
 
   ManifoldProperties props = manifold_get_properties(diff);
+  ManifoldProperties propscontext = manifold_get_properties(diffcontext);
 
   EXPECT_NEAR(props.volume, 0, 0.0001);
+  EXPECT_NEAR(propscontext.volume, 0, 0.0001);
```

Comment:
If what you're testing is translation, perhaps we should add a bounding box check to verify it moved.